actions/views/create_comment.tsx

```diff
@@ -34,11 +34,12 @@ import {GlobalState} from 'types/store';
 import {DispatchFunc, GetStateFunc} from 'mattermost-redux/types/actions';
 
 export function clearCommentDraftUploads() {
```

Comment:
I might move this around later to better consolidate the draft logic in one place, but I don't want to do that before the Global Drafts view PR is in